alibi/explainers/tests/test_shap_wrappers.py

```diff
@@ -999,6 +993,9 @@ def test_explain_kernel(monkeypatch, mock_kernel_shap_explainer, use_groups, sum
     assert explanation.meta.keys() == DEFAULT_META_KERNEL_SHAP.keys()
     assert explanation.data.keys() == DEFAULT_DATA_KERNEL_SHAP.keys()
 
+    # parameter keys
+    assert set(explanation.meta['params']) <= set(KERNEL_SHAP_PARAMS)
+
```

Comment:
So I guess it would be wise to test also if there are things in `explanation.meta['params']` that should not be there? In other words the difference between said object and `KERNEL_SHAP_PARAMS` should be empty because technically every member of  `explanation.meta['params']` should be specified in `KERNEL_SHAP_PARAMS`?

Reply:
This relates to my point whether we want strict subset equality? I'm sensing you are in favour so I'm happy to make the changes.

Reply:
`link` is never set for `KernelSHAP`

Reply:
Ok, that's an issue I'll correct in the patch. For some reason it has ended up in `data`.

Reply:
Regarding the inequality, I would guess we should check all the information we expect to be in `KERNEL_SHAP_PARAM` has been updated? So in a way I would check for equality, because what we want is to set the parameters we said we would in the schema. There is no point in putting a parameter in the schema if we're not going to set it and there is no point of adding data that is not defined in the schema.